src/App.js

```diff
@@ -1,23 +1,29 @@
 import React from 'react';
-import logo from './logo.svg';
+import { Router, Route, Switch } from 'react-router';
+import { createBrowserHistory } from 'history';
+import Header from './Components/Header/Header';
+import Footer from './Components/Footer/Footer';
+import Home from './Components/Home/Home';
+import Application from './Components/Application/Application';
+import Comparison from './Components/Comparison/Comparison';
 import './App.css';
 
+const browserHistory = createBrowserHistory();
+
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload (HI).
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <Header/>
+        <Router history={browserHistory}>
+          <Switch>
+            <Route exact={true} path="/" component={Home}></Route>
+            <Route exact={true} path="/submissions" component={Home}></Route>
+            <Route path="/submissions/:organizationId" component={Application}></Route>
+            <Route path="/comparisons/:organizationId" component={Comparison}></Route>
+          </Switch>
+        </Router>
+        <Footer/>
       </header>
```

Comment:
Do we want the header to wrap around the footer?

Reply:
Think I asked about this before, I believe we wanted to wrap everything in `<header>` for responsiveness (actual header is `Header` component)? I can definitely remove it if we want to go about it a different way though 🙂

Reply:
I think it is fine the way it is; just wanted to make sure it was intentional. 

It changes the way we define the menu/navbar css later on.